zomes/holo_acorn/code/src/lib.rs

```diff
@@ -480,6 +480,18 @@ mod holo_acorn {
             address,
         })
     }
+    #[zome_fn("hc_public")]
+    fn update_goal(goal_vote:GoalVote , address: Address) -> ZomeApiResult<GetResponse<Goal>> {
```

Comment:
Isn't there already an "update_goal" function? So maybe rename this one? That might be causing a bug